src/app/core/callendar/components/callendar.component.ts

```diff
@@ -148,6 +148,7 @@ export class CallendarComponent implements OnInit {
     }
 
     public eventClick(event) {
-        this.eventClickCallback.emit(this.events.find(element => element.id === event.detail.event.id));
+        let eventId = event.id;
```

Comment:
Is the eventId variable only used in line 152? If so, why not do this.eventClickCallback.emit(event.id) ?

Reply:
true, will fix